util/config.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
```

Comment:
Do we need these?

Reply:
No.  Vestigal from stuff I used to always do that I eventually decided I didn't need to do.